AmericanMedicalAssociation/AMA-style-guide
# EWL-3816 create assets branch

## Ticket(s)

Please do not submit a Pull Request without a relevant ticket! If you are creating a new pattern or feature, create an issue describing the need for this feature Github **first** and then link to it below.

**Github Issue**

NA

**Jira Ticket**

- [EWL-3816: Topics | SG | How to see off-release SG work in Drupal](https://issues.ama-assn.org/browse/EWL-3816)


## Description

This adds an option to the deploy task (also the option to include other options) to push to a new `dev-assets` branch.

## To Test

- ~`npm install` to install the minimist library which allows gulp to easily digest CLI options~
- Check out README.md for instructions on how to push to the new branch.
- Try it out. 

## Relevant Screenshots/GIFs

After a successful push you should see something like so:
<img width="591" alt="banners_and_alerts_and_styleguide_ _-bash_ _89x24" src="https://user-images.githubusercontent.com/238201/28686976-a6e8dd82-72d2-11e7-9a77-2b4f29f24c7d.png">

## Remaining Tasks

No

## Additional Notes

This relates to https://github.com/AmericanMedicalAssociation/ama-d8/pull/14
